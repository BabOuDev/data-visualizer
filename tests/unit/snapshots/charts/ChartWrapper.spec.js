import {shallowMount} from '@vue/test-utils';
import ChartWrapper from '@/components/charts/ChartWrapper.vue';

describe('ChartWrapper.vue', () => {
  it('should renders correctly', () => {
    const component = shallowMount(ChartWrapper, {
      props: {
        rows: [
          {colName: 'A'},
          {colName: 'B'},
          {colName: 'B'},
          {colName: 'C'},
          {colName: 'C'},
          {colName: 'C'},
        ],
        col: {path: 'colName'},
      },
    });
    expect(component.html()).toMatchSnapshot();
  });
});
