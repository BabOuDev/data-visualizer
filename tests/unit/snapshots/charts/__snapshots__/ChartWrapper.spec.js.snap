// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartWrapper.vue should renders correctly 1`] = `
<div class="container">
  <div class="buttons"><button class="pie active" title="Pie"><i></i></button><button class="bar" title="Bar"><i></i></button></div>
  <div class="chart-container">
    <pie-chart-stub rows="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]" col="[object Object]"></pie-chart-stub>
  </div>
</div>
`;
